bots/discord/pixelbot/egg-pixel-bot.json

```diff
@@ -1,13 +1,19 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1"
+        "version": "PTDL_v2"
     },
     "exported_at": "2020-01-19T11:11:24-05:00",
     "name": "pixel-bot",
     "author": "parker@parkervcp.com",
     "description": "A Discord bot written in Python using discord.py\r\n\r\nhttps:\/\/github.com\/Ispira\/pixel-bot",
-    "image": "quay.io\/parkervcp\/pterodactyl-images:alpine_python-3",
```

Comment:
pretty sure this is just a single image

Reply:
The egg only really mentions 3.6/3.7 so I was unsure, `~>3.6` versions should work with it anyway but I've removed them for now.